File: 453-minimum-moves-to-equal-array-elements/minimum-moves-to-equal-array-elements.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minMoves = function(nums) {
    let low = Math.min(...nums);
    let ans = 0;
    nums.forEach((ele) =>{
        ans += ele-low;
    })
    return ans;
};

File: 453-minimum-moves-to-equal-array-elements/minimum-moves-to-equal-array-elements.java
class Solution {
    public int minMoves(int[] nums) {
        int low = Integer.MAX_VALUE;
        for(int i : nums) low = Math.min(low,i);
        int ans = 0;
        for(int i : nums) ans += i-low;
        return ans;
    }
}